charts/oes/templates/configmaps/oes-datascience-config.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
```

Comment:
{{- if or (eq .Values.installationMode "OES-AP") (eq .Values.installationMode "EASY") -}} condition is missing here